Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml

```diff
@@ -289,6 +290,40 @@ script:
     description: Returns the schema of the index to fetch from. This commmand should be used for debugging purposes.
   - name: es-integration-health-check
     description: Returns the health status of the integration. This commmand should be used for debugging purposes.
+  - name: es-get-indices-statistics
+    description: Returns Elasticsearch indices statistics and information. This command is not supported for client type OpenSearch.
+    arguments:
+    - description: The maximum number of indices to return.
+      name: limit
+      required: false
+      defaultValue: '50'
+    - description: Whether to retrieve all the Elasticsearch indices. If true, the "limit" argument will be ignored.
+      name: all_results
+      required: false
+      defaultValue: false
+      auto: PREDEFINED
+      predefined:
+      - 'false'
+      - 'true'
+    outputs:
+    - contextPath: Elasticsearch.IndexStatistics.Name
+      description: The Name of the index.
+      type: String
+    - contextPath: Elasticsearch.IndexStatistics.Status
+      description: The Status of the index.
+      type: String
+    - contextPath: Elasticsearch.IndexStatistics.Health
+      description: The Health status of the index.
+      type: String
+    - contextPath: Elasticsearch.IndexStatistics.UUID
+      description: The UUID of the index.
+      type: String
+    - contextPath: Elasticsearch.IndexStatistics.Documents Count
```

Comment:
Is a period or underscore missing in the context path? Or is there an extra space in Documents Count?

Reply:
This is an extra space, I'll fix that, thank you!